velox/functions/prestosql/aggregates/ArbitraryAggregate.cpp

```diff
@@ -39,6 +39,10 @@ class ArbitraryAggregate : public SimpleNumericAggregate<T, T, T> {
     return sizeof(T);
   }
 
+  int32_t accumulatorAlignmentSize() const override {
```

Comment:
This needs to return the size of 128bit. And you need only to override it for ArbitraryAggregate<> where the types are INT128.